sys-auth/libyubikey/libyubikey-1.13.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="Low level C SDK for the Yubico YubiKey authentication device"
+DESCRIPTION="This package make up the low-level C software development kit for the Yubico YubiKey authentication device."
+HOMEPAGE="https://developers.yubico.com/yubico-c/"
+COPYRIGHT="2008-2015 Yubico AB"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://developers.yubico.com/yubico-c/Releases/libyubikey-1.13.tar.gz"
+CHECKSUM_SHA256="04edd0eb09cb665a05d808c58e1985f25bb7c5254d2849f36a0658ffc51c3401"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libyubikey$secondaryArchSuffix = $portVersion
+	cmd:modhex$secondaryArchSuffix = $portVersion
+	cmd:ykparse$secondaryArchSuffix = $portVersion
+	cmd:ykgenerate$secondaryArchSuffix = $portVersion
+	lib:libyubikey$secondaryArchSuffix = 0.1.7
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libusb$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libyubikey${secondaryArchSuffix}_devel = $portVersion
+	devel:libyubikey$secondaryArchSuffix = 0.1.7
+	"
+
+REQUIRES_devel="
+	libyubikey$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	lib:libcurl$secondaryArchSuffix
+	lib:libusb$secondaryArchSuffix
+"
+
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	runConfigure ./configure --disable-dependency-tracking
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	rm $libDir/libyubikey.la
+	prepareInstalledDevelLibs libyubikey
+	fixPkgconfig
+	packageEntries devel \
+			$developDir
```

Comment:
Can you please add a few extra changes to INSTALL?
This is the layout we like to have:
```
INSTALL()
{
	make install

	rm $libDir/libyubikey.la

	prepareInstalledDevelLib libyubikey
	fixPkgconfig

	if [ -n "$secondaryArchSuffix" ]; then
		rm -rf $manDir
	fi

	packageEntries devel \
		$developDir
}
```
The main change here is that we remove $manDir if we are building for a secondary arch.

Notice that I replaced prepareInstalledDevel**Libs** by prepareInstalledDevel**Lib**.
This is not required but it is welcome, since we only have one lib here.

BTW, in my previous review I should have mentioned that I was not willing to drop the "make install". Actually, we cannot control the number of lines GitHub puts in the context, so I should have copied the whole INSTALL ;-)
Hopefully you took the right decision, keeping it ;-)